src/app/components/OfflineBanner/hook.ts

```diff
@@ -5,9 +5,11 @@ import { AppError, AppErrors } from '../../../types/errors'
 import { useFormattedTimestampString } from '../../hooks/useFormattedTimestamp'
 import { paraTimesConfig } from '../../../config'
 
-export const useIsApiOffline = (network: Network): boolean => {
+export const useIsApiOffline = (network: Network): false | 'userOffline' | 'apiOffline' => {
```

Comment:
At this point I think we should rename this to `useNetworkProblem()`.

Reply:
I think that's too confusing: user's "network" && mainnet "network"?

Reply:
> isApiReachable?

No, that would make the "false" return value confusing.

> `useConnectionProblem()` ?

Reply:
but it's not necessarily a connection problem if API is unresponsive :D

Reply:
OK, I am fine with `useIsApiReachable`, but then we need a different result structure. `{ reacheble: true }` when all is fine, and  `{ reachable: false, reason: "whatever" }` in case of issues.